src/ExpenseTracker.Core/Repositories/Interface/IWorkpacseRepository.cs

```diff
@@ -8,5 +9,7 @@ public interface IWorkspaceRepository : IGenericRepository<Workspace>
     {
         Task<Workspace> GetDefaultWorkspace();
         Task<Workspace> GetByToken(string token);
+
+        Task<List<Workspace>> GetActiveWorkspaces(User user);
```

Comment:
@BirajMainali It might be better if we used `Id` instead of the entity here. It gives us more options.